src/pages/cookie-preferences.mdx

```diff
@@ -61,8 +61,16 @@ The below list details the cookies used in our website.
       <td>Set by Google to distinguish users.</td>
     </tr>
     <tr>
-      <td>_sp5_</td>
-      <td>The _sp5_ cookie, installed by Snowplow Analytics, stores information on how visitors use a website, while also creating an analytics report of the website's performance. Some of the data that are collected include the number of visitors, their source, and the pages they visit anonymously.</td>
+      <td>_sp5_id</td>
+      <td>The _sp5_id cookie stores user information that is created when a user first visits a site and updated on subsequent visits. It is used to identify users and track the users activity across a domain. This cookie stores a unique identifier for each user, a unique identifier for the users current session, the number of visits a user has made to the site, the timestamp of the users first visit, the timestamp of their previous visit and the timestamp of their current visit, references to previous session and first event in the current session, and index of the last event in the session.</td>
+    </tr>
+    <tr>
+      <td>_sp5_ses</td>
+      <td>The _sp5_ses cookie is used to identify if the user is in an active session on a site or if this is a new session for a user (i.e. cookie doesn't exist or has expired).</td>
+    </tr>
+    <tr>
+      <td>sp</td>
+      <td>The sp cookie stores a server-side collector generated unique identifier for a user that is sent with all subsequent tracking event events. Can be used as a first party cookie is the collector is on the same domain as the site.</td>
```

Comment:
I would just say “server-generated unique identified”, in case people don’t know what a collector is (although if they are reading these docs, they might)

Remove the sentence about first-party cookies, it’s not relevant for the reader